src/lucky_flow/driver.cr

```diff
@@ -1,7 +1,24 @@
 abstract class LuckyFlow::Driver
   @retry_limit : Time = 2.seconds.from_now
 
+  getter session : Selenium::Session { start_session }
+
   abstract def start_session : Selenium::Session
+  abstract def stop
+
+  def reset : Nil
+    @session.try &.cookie_manager.delete_all_cookies
+  end
+
+  def shutdown : Nil
+    @session.try &.delete
+    stop
+  end
+
+  def screenshot(path : String)
+    FileUtils.mkdir_p(File.dirname(path))
+    session.screenshot(path)
```

Comment:
should this also use the instance variable?

Reply:
No, the only places I used the instance variable was places where if it was never started, there's nothing to stop/reset